tableauserverclient/server/endpoint/projects_endpoint.py

```diff
@@ -20,6 +25,20 @@ def get(self, req_options=None):
         all_project_items = ProjectItem.from_response(server_response.content)
         return all_project_items, pagination_item
 
+    def get_default(self):
+        if (Version(self.parent_srv.version) < Version("2.7")):
+            for proj in Pager(self):
```

Comment:
Nit `Pager(self.parent_srv.projects)` is a little clearer than remembering what `self` is pointing to